common/src/db/embedded.rs

```diff
@@ -4,31 +4,40 @@ use postgresql_embedded::{PostgreSQL, Settings, VersionReq};
 use std::path::Path;
 use tracing::{info_span, Instrument};
 
-/// Create common default settings for the embedded database
-fn default_settings() -> anyhow::Result<Settings> {
+/// Create common default settings for the embedded database.
+/// Use the bool `temp` parameter to indicate if the pg_data
+/// directory is temporary or not.
+fn default_settings(temp: bool) -> anyhow::Result<Settings> {
```

Comment:
Passing a param to this function kind of betrays its name. :wink:

Reply:
LOL that's right :smile:

Reply:
> Can't we already achieve the goal of this PR with the EXTERNAL_TEST_DB env var? You could set DB_PORT to that of your "PM mode" instance and all the logs would be available to you in .trustify/data/start.log.

it works with `cargo test` without running the app ? :thinking: